Extension/src/LanguageServer/client.ts

```diff
@@ -1184,32 +1182,29 @@ export class DefaultClient implements Client {
                     await this.addTrustedCompiler(settings.defaultCompilerPath);
                     DefaultClient.updateClientConfigurations();
                     action = "confirm compiler";
-                    ui.ShowConfigureIntelliSenseButton(false, this);
+                    await ui.ShowConfigureIntelliSenseButton(false, this);
                 } else if (value === selectCompiler) {
-                    this.handleIntelliSenseConfigurationQuickPick(true, sender);
+                    await this.handleIntelliSenseConfigurationQuickPick(true, sender);
                     action = "show quickpick";
                 } else {
-                    this.showPrompt(selectCompiler, true, sender);
+                    await this.showPrompt(selectCompiler, true, sender);
```

Comment:
I think these `await` should be `void`. Same as in the promptSelectCompiler function above.